src/update_engine/omaha_hash_calculator.cc

```diff
@@ -170,10 +172,11 @@ bool OmahaHashCalculator::Base64Decode(const string& raw_in,
 bool OmahaHashCalculator::Finalize() {
   TEST_AND_RETURN_FALSE(hash_.empty());
   TEST_AND_RETURN_FALSE(raw_hash_.empty());
-  raw_hash_.resize(SHA256_DIGEST_LENGTH);
+  raw_hash_.resize(crypto_hash_sha256_BYTES);
+
   TEST_AND_RETURN_FALSE(
-      SHA256_Final(reinterpret_cast<unsigned char*>(&raw_hash_[0]),
-                   &ctx_) == 1);
+    crypto_hash_sha256_final(&hash_state_, 
+      reinterpret_cast<unsigned char*>(&raw_hash_[0])) == 0);
```

Comment:
Since C++11 vector has a `data()` member function that will get you a pointer to underlying memory.

```suggestion
      reinterpret_cast<unsigned char*>(raw_hash_.data())) == 0);
```